qutebrowser/config/configdata.yml

```diff
@@ -1274,20 +1276,20 @@ tabs.title.format:
       - current_url
     none_ok: true
   desc: |
-    The format to use for the tab title.
+    Format to use for the tab title.
     The following placeholders are defined:
 
-    * `{perc}`: The percentage as a string like `[10%]`.
-    * `{perc_raw}`: The raw percentage, e.g. `10`
-    * `{title}`: The title of the current web page
+    * `{perc}`: Percentage as a string like `[10%]`.
+    * `{perc_raw}`: Raw percentage, e.g. `10`
+    * `{title}`: Title of the current web page
```

Comment:
Those should have a trailing period for consistency, but I'll fix that up after merging.

Reply:
Fixed in c22a27d47f8f0f331ff61bc089d20d5b0ffbb416